index.html

```diff
@@ -4,83 +4,219 @@
+		</div>
+	</div>
+	<div class="main">
+		<div class="container value">
+			<div class="row justify-content-center">
+				<div class="col-lg-8 center">
+					<h1 class="pop"> For any creatives at any level to create and share their imagination in XR</h1>
+					<!--Toy Faces created by Amrit Pal Singh (https://amritpaldesign.com/toy-faces)-->
+				</div>
+				<div class="col-lg-12">
+					<img style="width:inherit" class="rounded shadow-lg" src="./images/artist2.png">
+				</div>
+
+			</div>
+		</div>
+		<div class="container value">
+			<div class="row pt-5">
+				<div class="col-lg-6">
+					<img class="demo" src="./images/creation.png" width="400">
+				</div>
+				<div class="col-lg-6">
+					<h1 class="pop">Simply create your XR story with just a few taps</h1>
+					<h3 class="motto">experience a unified and simplified creation process to unblock your creativity</h3>
+				</div>
+
+			</div>
+		</div>
+		<div class="container value">
+			<div class="row">
+				<div class="col-lg-6 order-md-2 order-sm-1">
+					<img class="demo" src="./images/sanbox.png" width="400">
+					<!--<a href='https://www.freepik.com/psd/woman'>Woman psd created by freepik - www.freepik.com</a>-->
+				</div>
+				<div class="col-lg-6  order-md-1 order-sm-2">
+					<h1 class="pop">The XR sandbox for your creativity</h1>
+					<h3 class="motto">powered by the community-built 3D assets, music, animation and effects library</h3>
+
+				</div>
+
+			</div>
+		</div>
+		<div class="container value">
+			<div class="row">
 				<div class="col-lg-6">
-					<video playsinline="" muted="" class="img-rounded video" src="images/demo.mp4" style="width:inherit" loop autoplay=""></video>
+					<img class="demo" src="./images/alldevice2.png" width="400">
 				</div>
+				<div class="col-lg-6">
+					<h1 class="pop">Share and enjoy many XR creations together in any device</h1>
+					<h3 class="motto">remove the barrier between your XR arts and your audience</h3>
 
+				</div>
 			</div>
+		</div>
+		<div class="end rounded shadow-lg">
+			<div class="container value mt-5">
+				<div class="row justify-content-center">
+					<div class="col-lg-9 center-close mt-5">
+						<h1 class="pop">Bring more people to contribute to the future of XR</h1>
+						<h3 class="motto">Our strength lies in the hands of our users, forming a mobile XR community welcoming everyone of all levels and backgrounds to share their creative story through art and music.
+						</h3>
+					</div>
+					<div class="col-lg-12">
+						<img style="width:inherit" src="./images/community.png">
+							<!--3D hands created by ThreeDee (https://www.handz.design/)-->
+					</div>
 
+				</div>
+			</div>
 		</div>
-		<div class="footer">
-			<div class="container">
-				© 2021 Virtision. All Rights Reserved.
 
-			</div><!-- /.navbar-collapse -->
+		<div class="container value mt-5">
+			<div class="row justify-content-center">
+				<div class="col-lg-12 center">
+					<h1 class="pop"> Be the first to get the app </h1>
```

Comment:
"Be one of the first to get the app"
"Get the app early"
"Be first to get the app"

Reply:
"Be one of the first to get the app"